src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -91,42 +104,43 @@ extends AbstractMap[Key, Value]
     table = new Array[Entry](newSize)
     mask = newSize - 1
     oldTable.foreach( entry =>
```

Comment:
Sadly. Though I haven't looked at the disassembled code with 2.12.

This call would also benefit from a streamlined `findIndex()`. Ditto in `remove()`.